controller/RestController.php

```diff
@@ -1002,4 +1002,44 @@ public function related($request)
         $ret = $this->transformPropertyResults($request->getUri(), $request->getLang(), $related, "related", "skos:related");
         return $this->returnJson($ret);
     }
+
+    /**
+     * Used for querying new concepts in the vocabulary
+     * @param Request $request
+     * @return object json-ld wrapped list of changed concepts
+     */
+    public function newConcepts($request)
+    {
+        return $this->changed($request);
+    }
+
+    /**
+     * Used for querying modified concepts in the vocabulary
+     * @param Request $request
+     * @return object json-ld wrapped list of changed concepts
+     */
+    public function modifiedConcepts($request)
+    {
+        return $this->changed($request, 'dc:modified');
+    }
+
+    /**
+     * Used for querying changed concepts in the vocabulary
+     * @param Request $request
+     * @return object json-ld wrapped list of changed concepts
+     */
+    private function changed($request, $prop='dc:created')
+    {
+        // set language parameters for gettext
+        //$this->setLanguageProperties($request->getLang());
+        $vocab = $request->getVocab();
+        $offset = ($request->getQueryParam('offset') && is_numeric($request->getQueryParam('offset')) && $request->getQueryParam('offset') >= 0) ? $request->getQueryParam('offset') : 0;
+        $contentLang = ($request->getContentLang() != null) ? $request->getContentLang() : $request->getLang();
```

Comment:
I don't think this method needs a separate contentLang setting, because as a REST method it doesn't really return any UI strings, just concept labels. So I'd just use `$request->getLang()` here for all language settings

Reply:
Yep, the existing method parameters looked really odd for this purpose. I will write a more general-purpose method for the REST API that takes only one language parameter and returns the modified concepts by timestamp, in contrast to the existing version.